### ISSUE TITLE
Errorbars in plots when using WSTAT

### ISSUE THREAD

**Issue by anetasie** [sherpa/sherpa#1301]
The calculation of the errorbars in `plot_fit` when using WSTAT needs to account for additional errors when subtracting the background.

**Comment by DougBurke** [comment]
We have significantly-more-serious errors than the error bars when plotting WSTAT data! e.g. #1114 which is a rework of #694

**Comment by anetasie** [comment]
@DougBurke Yes, there are more serious issues and use of WSTAT is a partial solution to the basic statistical issues in modeling Poisson data with background.
